node_launcher/node_set/tor.py

```diff
@@ -0,0 +1,43 @@
+import psutil
+
+from node_launcher.node_set.lnd import Lnd
+from node_launcher.services.configuration_file import ConfigurationFile
+from node_launcher.constants import (
+    IS_LINUX,
+    IS_MACOS,
+    IS_WINDOWS,
+    LND_DIR_PATH,
+    TOR_PATH, # Change to TOR_DIR_PATH
+    OPERATING_SYSTEM,
+    #BITCOIN_DATA_PATH,
+    #BITCOIN_CONF_PATH,
+    #TOR_DATA_PATH,
+    #TOR_TORRC_PATH,
+    #LND_CONF_PATH,
+    #TOR_EXE_PATH,
+)
+
+class Tor(object):
+    file: ConfigurationFile
+    software: TorSoftware
```

Comment:
Is this defined anywhere or just a "stub"?

Reply:
Just an incomplete stub for now. I'm just using this PR to visually see what changes I've made so far. It's not at all functional as yet 🙈